public/css/homepage.css

```diff
@@ -104,38 +104,97 @@ body {
 #filters {
   display: flex;
   flex-flow: row;
```

Comment:
Like said above, flex-flow: row is the default. Also, a flex-wrap is missing here, because the filters are not visible when the window is resized.

(and correct the space at line 17)